tests/backup/backup_helper.go

```diff
@@ -1497,12 +1502,34 @@ func DeletePodWithLabelInNamespace(namespace string, label map[string]string) er
 	}
 
 	// fetch the newest set of pods post wait for pods to come up
-	pods, err = core.Instance().GetPods(namespace, label)
+	if ignoreLabel {
+		nolabel := make(map[string]string)
+		pods, err = core.Instance().GetPods(namespace, nolabel)
+	} else {
+		pods, err = core.Instance().GetPods(namespace, label)
+	}
 	if err != nil {
 		return err
 	}
-
 	for _, pod := range pods.Items {
+		flag := false
+		if ignoreLabel {
+			for key, podlabel := range label {
+				for key2, podlabel2 := range pod.GetLabels() {
+					if key == key2 && podlabel2 == podlabel {
+						flag = true
+						break
+					}
+				}
+				if flag {
+					break
+				}
+			}
+			if flag {
+				continue
+			}
```

Comment:
```go
flag := false
for key, podlabel := range pod.GetLabels() {
    if podlabel2, exists := label[key]; exists && podlabel == podlabel2 {
        flag = true
        break
    }
}
if flag {
    continue
}
```
?

Reply:
if possible log pod name while skipping?

Reply:
Yes this looks better we'll cleanup

Reply:
Done